modules/core/package.json

```diff
@@ -44,8 +44,8 @@
     "gen-docs": "typedoc"
   },
   "dependencies": {
-    "@bitgo/account-lib": "^2.8.0-rc.2",
-    "@bitgo/statics": "^6.3.0-rc.1",
+    "@bitgo/account-lib": "^2.8.0-rc.3",
+    "@bitgo/statics": "^6.3.0-rc.2",
```

Comment:
Please do not modify the package.json versions for packages within this repository - these are managed by lerna as part of the release process.